R/plumb-block.R

```diff
@@ -130,7 +131,7 @@ plumbBlock <- function(lineNum, file){
       }
 
       if (!is.na(s) && !s %in% names(.globals$serializers)){
```

Comment:
consider one of the following
 - adding "all", "none" to the check here
 - removing the check here and letting the make_parsers inner check do it's job
 - make the check a function and use the same for make_parsers and here

Reply:
Added `registered_serializers()`. Going to leave the check against the `registered_parsers()` as `stopOnLine()` gives better context than what `make_parser()` can provide